2nd-gen/uppercase/README.md

```diff
@@ -0,0 +1,30 @@
+# Firebase SDK for Cloud Functions 2nd Gen Quickstart - Realtime Database
+
+This quickstart demonstrates using **Firebase SDK for Cloud Functions** setup with a **Firebase database**.
```

Comment:
Suggested "using **Firebase SDK for Cloud Functions** with with **Firebase Realtime Database**.